templates/default/src/pages/index/index.jsx

```diff
@@ -1,20 +1,25 @@
 import React from 'react'
 import Link from 'next/link'
-import GlobalHead from '../../components/organisms/GlobalHead'
+import PageTitle from '../../components/organisms/PageTitle'
 
 const Home = () => {
   return (
-    <>
-      <GlobalHead title="%%APPNAME%%" />
+    <div className="text-center">
+      <PageTitle title="%%APPNAME%%" />
       <p>Welcome to the boilerplate React app using NextJS!</p>
       <p>
         Try navigating to
-
         <Link href="/example/second-page">another page</Link>
-
         and observe the URL changing
       </p>
-    </>
+      <style jsx global>
```

Comment:
since this is on both the second page and first page it would be recommended to do a Layout component, but for our purposes, not everyone would that that usecase so I think it's ok we duplicate it here.